File: Niveau 2/2 – Découverte des tableaux/06 - Visite de la mine.py
totalDep = int(input())
dep = [0] * totalDep
depHelper = [0, 2, 1, 3, 5, 4]
for i in range(totalDep):
    dep[i] = int(input())
for i in range(totalDep-1, -1, -1):
    print(depHelper[dep[i]])

File: Niveau 2/2 – Découverte des tableaux/06 - Visite de la mine.cpp
#include <iostream>
using namespace std;
int main()
{
  int nbDep;
  cin >> nbDep;
  int dep[nbDep] = {0};
  int depHelper[5] = {2, 1, 3, 5, 4};
  for (int i = 0; i < nbDep; i++)
  {
    int d;
    cin >> d;
    dep[i] = d;
  }
  for (int i = nbDep - 1; i >= 0; i--)
  {
    cout << depHelper[dep[i] - 1] << endl;
  }
}